bundles/org.openhab.binding.linky/src/main/java/org/openhab/binding/linky/internal/api/ExpiringDayCache.java

```diff
@@ -111,7 +111,7 @@ public final synchronized void invalidateValue() {
      * @return true if the value is expired
      */
     public boolean isExpired() {
-        return !LocalDateTime.now().isBefore(expiresAt);
+        return LocalDateTime.now().isAfter(expiresAt);
```

Comment:
I hope you are sure about this change ;)
Maybe the idea was to handle the case of equal times.

Reply:
Reverted